### DISCUSSION TITLE
Anzeige Darstellungsvariante

### DISCUSSION THREAD

**Discussion by thomaskofler** [e-netze/webgis-community#99] [General]
Habe einen kleinen "Bug" entdeckt und zwar, wenn man bei der Darstellungsvariante eine Untergruppe einfügt und dann bei einem Layer in der Untergruppe einen Link-Button erstellt, dann wird das Ein/Aus-Kästchen bei der Untergruppe entfernt. Siehe folgendes Beispiel, FLS kein Info/Link-Button, ÖWG mit Link-Button:

![grafik](https://github.com/gis-eni/webgis-forum/assets/119412172/1143615c-038c-4fc2-a62f-6ff98a6c3de1)

**Comment by gis-eni** [comment]
Mit Build 3601 behoben
